### ISSUE TITLE
Redundant bracket warning for unary unboxed tuple

### ISSUE THREAD

**Issue by sjakobi** [ndmitchell/hlint#1350]
The current `master` (9f18b433b3c7f4e7976aa3983289edbcf2e0d84b) incorrectly warns about redundant brackets on unary unboxed tuples in `Data.HashMap.Internal`, e.g. [here](https://github.com/haskell-unordered-containers/unordered-containers/blob/42a25dbc19babf7c1153ae19bdef609f8308de04/Data/HashMap/Internal.hs#L970):

```
Data/HashMap/Internal.hs:970:59-62: Warning: Redundant bracket
Found:
  (v')
Perhaps:
  v'
```

I've haven't been able to reproduce the issue on a smaller example though, so maybe `hlint` is getting confused by some aspect of that module.

**Comment by zliu41** [comment]
@sjakobi I think (based on the column number) what HLint warns is the `| otherwise -> Leaf h (L k (v'))` on line 971, which is a legitimate warning. Perhaps you are using a different version of `Internal.hs` where that line is 970?

**Comment by sjakobi** [comment]
Oh, yes! :man_facepalming: Sorry for the noise, and thanks for the help!
